cortex/blender/blendlib.py

```diff
@@ -75,14 +88,14 @@ def add_vcolor(hemis, mesh=None, name='color'):
         if len(mesh.vertices) == len(hemis[1]):
             color = hemis[1]
 
-    vcolor = mesh.vertex_colors.new(name)
+    vcolor = mesh.vertex_colors.new(name=name)
     if hasattr(mesh, "loops"):
         loopidx = [0]*len(mesh.loops)
         mesh.loops.foreach_get('vertex_index', loopidx)
 
         if not isinstance(color[0], (list, tuple)):
             for i, j in enumerate(loopidx):
-                vcolor.data[i].color = [color[j]]*3
+                vcolor.data[i].color = [color[j]]*3 + [1]
```

Comment:
I guess this is adding an alpha channel.
Is this change only for 2.80+ ?
Is it backward compatible ?